test/integration_test/stork-specs/stork-webhook.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: stork-webhooks-cfg
+webhooks:
+  - name: stork.webhook.com
+    failurePolicy: Ignore
+    clientConfig:
+      service:
+        name: stork-service
+        namespace: kube-system
+        path: "/mutate"
+      caBundle: ""
+    rules:
+      - operations: [ "CREATE" ]
+        apiGroups: ["apps", ""]
+        apiVersions: ["v1"]
+        resources: ["deployments","statefulset"]
```

Comment:
nit: missing newline

Reply:
Removed the specs as they are created temporarily during the build.